src/Listeners/SendConfiguredNotifications.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace DoubleThreeDigital\SimpleCommerce\Listeners;
+
+use DoubleThreeDigital\SimpleCommerce\Events\OrderPaid;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Notification;
+use Illuminate\Support\Str;
+use ReflectionClass;
+
+class SendConfiguredNotifications implements ShouldQueue
+{
+    public function handle(OrderPaid $event)
+    {
+        $eventName = Str::of(get_class($event))
+            ->afterLast('\\')
+            ->snake()
+            ->__toString();
+
+        $notifications = collect(Config::get('simple-commerce.notifications'))->get($eventName);
+
+        foreach ($notifications as $notification => $config) {
+            $freshNotification = null;
+
+            // $reflection = new ReflectionClass($notification);
+            // $constructor = $reflection->getConstructor();
+
+            $notifiables = $this->getNotifiables($config, $notification, $event);
+            $notification = new $notification($event->order);
```

Comment:
Last thing before adding the new triggers - we need to figure out how to pass in the right parameters from the event to the notification, based on what the notification's constructor parameters are. 

We could do some clever guessing using reflection magic. Just need to figure out how to pass an array of guesses in...